hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/tools/DFSHAAdmin.java

```diff
@@ -262,8 +262,13 @@ private int transitionToObserver(final CommandLine cmd)
     if (!checkManualStateManagementOK(target)) {
       return -1;
     }
-    HAServiceProtocol proto = target.getProxy(getConf(), 0);
-    HAServiceProtocolHelper.transitionToObserver(proto, createReqInfo());
+    try {
+      HAServiceProtocol proto = target.getProxy(getConf(), 0);
+      HAServiceProtocolHelper.transitionToObserver(proto, createReqInfo());
+    } catch (IOException e) {
+      errOut.println("TransitionToObserver failed! " + e.getMessage());
```

Comment:
minor nit: `TransitionToObserver` -> `transitionToObserver` for consistency with other prints e.g. L254

Also `getMessage()` -> `getLocalizedMessage()`